libraries/stdlib/samples/test/samples/collections/collections.kt

```diff
@@ -1033,4 +1033,69 @@ class Collections {
         }
 
     }
+
+    class BinaryOperations {
+
+        @Sample
+        fun unionOfGeneric() {
+            val unionOfArrayAndIterable = arrayOf("Apple", "Banana", "Orange", "Apple").union(
```

Comment:
I'd suggest to specify the explicit type on the result variable (here and below), as it may be unclear from the sample code.

Reply:
done

Reply:
> here and below

I mean in all union samples.

Reply:
> > here and below
> 
> I mean in all union samples.

done